### PR TITLE
Add data source in datacollection

### PR DESCRIPTION
**Pull request by shiyu22**
Signed-off-by: shiyu22 <shiyu.chen@zilliz.com>

### CODE REVIEW

**Review by reiase — CHANGES_REQUESTED** [review]

File: towhee/functional/data_collection.py [hunk 1]
```diff
@@ -262,6 +262,19 @@ def safe(self):
         """
         return self.exception_safe()
 
+    def output(self, name: str = 'feature_vector'):
```
[the remark this answers is not in the document]
Comment (shiyu22) [reply towhee/functional/data_collection.py:265]: good idea

File: towhee/utils/pil_utils.py [hunk 2]
```diff
@@ -44,6 +48,36 @@ def from_src(src: Union[str, PosixPath]) -> Image:
     return img
 
 
+def from_zip(zip_src: Union[str, PosixPath], pattern: str = '*.JPEG') -> Image:
```
[the remark this answers is not in the document]
Comment (shiyu22) [reply towhee/utils/pil_utils.py:51]: One is with pillow and another is cv2, but Pillow will get an error with 3 channel image, so will reserve cv2 method.

**Review by reiase — APPROVED** [review]

**Pull request merged**